adsingestp/parsers/copernicus.py

```diff
@@ -138,6 +122,9 @@ def _parse_pubdate(self):
             self.base_metadata["pubdate_electronic"] = self.input_metadata.find(
                 "publication_date"
             ).get_text()
+            self.base_metadata["pubdate_print"] = self.input_metadata.find(
+                "publication_date"
```

Comment:
This is finding the same node as for the `pubdate_electronic` - I'm guessing it should be finding a different node?

Reply:
There is only one pubdate given from this publisher even though they have print and online versions

Reply:
Hmm, ok, let's ask @seasidesparrow what he prefers here. (Not sure if tagging him here will work - you may need to ask on Slack)

Reply:
Carolyn said on slack that she would prefer both electronic and print pubdates to be populated.